examples/with-typescript-graphql/schema/type-defs.graphqls

```diff
@@ -0,0 +1,9 @@
+type User {
```

Comment:
Why is the file extension `.graphqls` when it could be `.graphql`?

Reply:
It's to differentiate GraphQL **schema** from GraphQL **documents** so that webpack can use `graphql-tag/loader` for schema and `graphql-let/loader` for documents. And another reason for `.graphqls` is that [the core library supports it](https://github.com/ardatan/graphql-toolkit/blob/d29e518a655c02e3e14377c8c7d3de61f08e6200/packages/loaders/graphql-file/src/index.ts#L9) as schema files. 

Yes, if we target the specific directory for each, we can use the same extensions as below, but what do you think? I thought it can lead to more pitfalls for users.

```diff
         test: /\.graphql$/,
+        include: path.join(__dirname, 'pages'),
...
-        test: /\.graphqls$/,
+        test: /\.graphql$/,
+        include: path.join(__dirname, 'schema'),
```